src/kolibri_tools/utils.py

```diff
@@ -6,6 +6,20 @@
 from config import KOLIBRI_PORT
 
 
+def get_key_drive():
+    # Add fallback content from endless key removable device
+
+    from kolibri.core.discovery.utils.filesystem import enumerate_mounted_disk_partitions
+    drives = enumerate_mounted_disk_partitions()
+    key_drive = None
+    for path, drive in drives.items():
+        if drive.name == 'eoslive':
```

Comment:
I think it would be nice if we do a quick `os.path.exists(drive.datafolder)` here as well, just to be sure.

Reply:
Good idea.